integration-tests/src/server.js

```diff
@@ -20,26 +21,44 @@ const app = express()
 const server = http.createServer(app)
 server.setTimeout(serverTimeout)
 
+const firestore = new Firestore()
+
 const testTimeout = async timeout => {
   await delay(timeout)
   throw new Error(`Test timeout after ${timeout}ms`)
 }
 
+const reportTest = (testGroup, startTime, host, testName, errorMessage) => {
+  const endTime = new Date()
+  const testReport = { testGroup, runtimeInMilliseconds: endTime - startTime, status: errorMessage ? 'error' : 'success', testName, errorMessage, host }
+  firestore.collection('tests').doc().create(testReport)
+}
+
 const runTest = fn => withPuppeteer(async ({ browser, context, name, page, req, timeout }) => {
   const targetEnvParams = { ...envs[req.query.targetEnv] }
+  const testGroup = req.query.testGroup
+  const startTime = new Date()
+  const testName = req.route.path
```

Comment:
This will give the full path, e.g. `'/test/foo'`, right? I think the fact that we're serving requests at `/test` is a current implementation detail, and not something that we should persist to the DB. What about just using `name`?

Reply:
Yes, this will give the full path. I will change.